keps/sig-api-machinery/5073-declarative-validation-with-validation-gen/kep.yaml

```diff
@@ -0,0 +1,46 @@
+title: Declarative Validation
+kep-number: 5073
+authors:
+  - "@aprindle"
+  - "@yongruilin"
```

Comment:
If there are no objections, I'd like to claim co-authorship on this one, which precludes me from being an approver, but for all the right reasons.

Reply:
Added you as a co-author, updated this to be:
```
authors:
  - "@aprindle"
  - "@yongruilin"
  - "@jpbetz"
```